website/content/en/docs/deployment/vanilla/guide.md

```diff
@@ -150,61 +150,76 @@ well.
 Install Istio resources:
 
 ```sh
-kustomize build common/istio-1-9/kubeflow-istio-resources/base | kubectl apply -f -
+kustomize build upstream/common/istio-1-9/kubeflow-istio-resources/base | kubectl apply -f -
 ```
 
 #### Kubeflow Pipelines
 
 Install the [Multi-User Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/multi-user/) official Kubeflow component:
 
 ```sh
-kustomize build apps/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user | kubectl apply -f -
+kustomize build upstream/apps/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user | kubectl apply -f -
 ```
 
-#### KFServing
+#### KServe / KFServing
 
-Install the KFServing official Kubeflow component:
+KFServing was rebranded to KServe.
+
+Install the KServe component:
+
+```sh
+kustomize build upstream/contrib/kserve/kserve | kubectl apply -f -
```

Comment:
why is this not using `awsconfigs/apps/kserve`? is it because that overlay is only for ALB ingress?